system/mrm_handler/README.md

```diff
@@ -0,0 +1,44 @@
+# mrm_handler
+
+## Purpose
+
+MRM Handler is a node to select a proper MRM from a system failure state contained in OperationModeAvailability.
+
+## Inner-workings / Algorithms
+
+### State Transitions
+
+![mrm-state](image/mrm-state.svg)
+
+## Inputs / Outputs
+
+### Input
+
+| Name                                   | Type                                                 | Description                                                                                         |
+| -------------------------------------- | ---------------------------------------------------- | --------------------------------------------------------------------------------------------------- |
+| `/localization/kinematic_state`        | `nav_msgs::msg::Odometry`                            | Used to decide whether vehicle is stopped or not                                                    |
+| `/system/operation_mode/availability`  | `autoware_adapi_v1_msgs::msg::OperationModeState`    | Used to select proper MRM from system available mrm behavior contained in operationModeAvailability |
```

Comment:
The type seems to be wrong.

Reply:
I fixed it. Thanks for your recommendation!